app/views/notifier/order_receipt.html.haml

```diff
@@ -1,29 +1,30 @@
 - @order.order_details.each { |od| od.send(:extend, PriceDisplayment) }
 
-%p= t('.intro')
+%p= t(".intro")
 
 %ul
   %li
-    %label Ordered at
-    =h @order.ordered_at.strftime("%b %e, %Y %l:%M %p")
+    %label= t(".label.ordered_at")
+    = l(@order.ordered_at, format: :receipt)
   %li
-    %label Facility
-    =h @order.facility
+    %label= t(".label.facility")
+    = @order.facility
   %li
-    %label Ordered By
-    =h "#{@order.created_by_user.full_name}"
+    %label= t(".label.ordered_by")
+    = @order.created_by_user.full_name
 
 %table.table.table-striped.table-hover
   %thead
     %tr
-      %th Order #
-      %th Product
-      %th Quantity
-      %th Status
+      %th= t(".th.order_number")
+      %th= t(".th.product")
+      %th= t(".th.quantity")
```

Comment:
Would some of these be better as `OrderDetail.human_attribute_name(:quantity)` as opposed to specifically targeting this view?